knowledge-base/grid-filterrow-bool-customization.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Change the Boolean Filtering Options in a FilterRow Dropdown List
```

Comment:
"Drop down" are normally two separate words. You can also remove "List", and consider "Filter Row", depending on what words customers used in their tickets.